src/app/pdp/directives/ot-social-links.html

```diff
@@ -0,0 +1,31 @@
+<!-- social links -->
+<div class="social">
+    <div id="social" class="clearfix">
+        <p class="share-this">Share this item:</p>
+
+        <div class="google-plus" ng-if="showGooglePlus">
+            <g:plusone size="medium" annotation="none"></g:plusone>
+        </div>
+
+        <div class="tweet-btn" ng-if="showTwitter">
+            <a href="http://twitter.com/share" class="twitter-share-button" data-url="http://knickknackrecords.com/products/a-ha-scoundrel-days-cassette" data-count="none" data-via="shopify">Tweet</a><script type="text/javascript" src="//platform.twitter.com/widgets.js"></script>
+        </div>
+
+        <div class="pinterest" ng-if="showPinterest">
+            <a href="http://pinterest.com/pin/create/button/?url=http://knickknackrecords.com/products/a-ha-scoundrel-days-cassette&media=//cdn.shopify.com/s/files/1/0267/3701/products/A-ha_Scoundrel_Days_large.jpg?v=1461628633" class="pin-it-button" count-layout="horizontal">Pin It</a>
+            <script type="text/javascript" src="//assets.pinterest.com/js/pinit.js"></script>
+        </div>
+
+        <div class="facebook-like" ng-if="showFacebook">
+            <iframe src="//www.facebook.com/plugins/like.php?href=http://knickknackrecords.com/products/a-ha-scoundrel-days-cassette&amp;send=false&amp;layout=button_count&amp;width=120&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;font&amp;height=21" scrolling="no" frameborder="0" style="border:none; overflow:hidden; width:120px; height:21px;" allowTransparency="true"></iframe>
+        </div>
+    </div>
+
+    <script type="text/javascript">
```

Comment:
@igorGalchenko move this script inside `google-plus` section, so it will not be created when google-plus button inactive